docusaurus/README.md

```diff
@@ -5,7 +5,7 @@ static website generator.
 
 ### Requirements
 
-* NodeJS 14+
+* NodeJS 16.14+
```

Comment:
Do we need to update .github/workflows/deploy-gh-pages.yml as well? (currently `16.x`, should be `>=16.14`?)

Reply:
Good catch, thanks!